src/app/homepage/homepage.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+  <section class="s1">
+    <div class="myinfodiv">
+      <app-infobox [boxcontent]="boxdata[0]"></app-infobox>
+    </div>
+
+    <div class="animdiv">
+      <section>
+        <div class="kframediv">
+          <span
+            >-LATEST WORK AND FEATURED -LATEST WORK AND FEATURED -LATEST WORK
```

Comment:
Please make sure all texts are in the `component` and you use variables to render them here